explorer/src/components/account/StakeAccountSection.tsx

```diff
@@ -129,17 +133,19 @@ function OverviewCard({
 function DelegationCard({
   stakeAccount,
   stakeAccountType,
+  activation,
 }: {
   stakeAccount: StakeAccountInfo | StakeAccountWasm;
   stakeAccountType: StakeAccountType;
+  activation: StakeActivationData | null;
 }) {
   const displayStatus = () => {
     // TODO check epoch
```

Comment:
This comment can be removed now. It was referring to calculating whether a stake account was still active or not